monitoring/config-generator/src/generate.js

```diff
@@ -56,6 +56,16 @@ function generate () {
         describe: 'external URL for the promtheus service',
         default: 'http://localhost:9090',
         type: 'string'
+      },
+      'cadvisor-port': {
+        describe: '[docker compose mode only] the port defined for cadvisor',
```

Comment:
Any particular reason this would only work with docker compose? I know it wouldn't work with our Chicago deployment because we don't have cadvisor or node-exporter, but it could work with other k8s deployments if configured properly, maybe?

Reply:
you're right, but since the way `cadvisor` is configured in `docker-compose` vs `kube` setups, i've added another flag, `--kube-cadvisor` to enable that.